charts/trino/templates/configmap-jmx-exporter.yaml

```diff
@@ -1,13 +1,15 @@
-{{- if .Values.jmx.exporter.enabled -}}
+{{- with .Values.jmx.exporter }}
```

Comment:
This requires more changes in lines that don't reference this value, revert it.

Reply:
I reverted this change.